tool/src/test/resources/org/datacommons/tool/genmcf/measurementresult/output/summary_report.html

```diff
@@ -53,6 +53,8 @@
     </div>
     <h1>Summary Report</h1>
     <h3>Table of Contents</h3>
+    <button onclick="open_all_details()">Expand All</button>
+    <button onclick="close_all_details()">Collapse All</button>
```

Comment:
puzzled by this change?

Reply:
put the comment in the wrong place, see thread.

also note that this preview on the GitHub UI links to "changes from 7 commits" (from your last review), not the entirety of the PR.